src/main/scala/inox/Options.scala

```diff
@@ -232,3 +232,5 @@ object optSelectedSolvers extends SetOptionDef[String] {
 }
 
 object optNoColors extends inox.FlagOptionDef("no-colors", false)
+
+object optNonIncremental extends inox.FlagOptionDef("no-incremental", false)
```

Comment:
Shouldn't this be somewhere under solvers?

Reply:
I ended up removing the option to use `no-inc:solver` instead